code/parachain/frame/pablo/src/lib.rs

```diff
@@ -252,7 +253,7 @@ pub mod pallet {
 			+ Ord;
 
 		/// Type representing the Balance of an account.
-		type Balance: BalanceLike + SafeSub + Zero + FixedPointOperand;
+		type Balance: BalanceLike + SafeSub + Zero + FixedPointOperand + Into<u128>;
```

Comment:
I originally envisioned importing Pablo from pallet ibc. Or this can be done totally generic and wired up at the config level of the runtime instead

Reply:
@RustNinja this is why I added to composbale-traits package fee converter trait

Reply:
Yeah this is something we discussed and I probably gave him the wrong directions

Reply:
Tbh it also makes sense to define that trait in Centauri because it is where it is used

Reply:
one more argument to declare FlatFeeConvertor trait to centauri is that

composable-trait uses some features from nightly channel.
i faced with  a lot of errors that it not possible anymore to build/test centauri palelt-ipc without +nightly channel if we will bring composable trait to centauri.
of course we can add toolchain channel
`channel = "nightly-2022-11-17"` 
it will works . but anyway it is one more dependency...

Reply:
Cool then let’s keep it in Centauri